src/modules/Dashboard/features/Respondents/Popups/EditRespondentPopup/EditRespondentPopup.tsx

```diff
@@ -23,14 +24,19 @@ export const EditRespondentPopup = ({
 }: EditRespondentPopupProps) => {
   const { t } = useTranslation('app');
   const dispatch = useAppDispatch();
+  const isTeamMember = chosenAppletData?.subjectTag === 'Team';
 
   const [isServerErrorVisible, setIsServerErrorVisible] = useState(true);
 
   const onCloseHandler = (shouldRefetch = false) => onClose(shouldRefetch);
 
   const { handleSubmit, control, setValue, getValues, trigger } = useForm<EditRespondentForm>({
     resolver: yupResolver(editRespondentFormSchema()),
-    defaultValues: { secretUserId: '', nickname: '' },
+    defaultValues: {
+      secretUserId: chosenAppletData?.respondentSecretId ?? '',
+      nickname: chosenAppletData?.respondentNickname ?? '',
+      tag: chosenAppletData?.subjectTag ?? ('' as const),
```

Comment:
Oh neat… of course you can do `as const` in general in these types of cases, instead of `as ParticipantTag`. Nice tip. :+1:

Reply:
Yeah — I didn't think it was significant enough to call out when reviewing your other PR, because either approach is totally valid, but not requiring import statements this way is one reason I like unions like these over enums where possible 👍.